src/components/scrim/edit-scrim-form.tsx

```diff
@@ -293,13 +314,32 @@ export function EditScrimForm({
           <FormItem>
             <FormLabel>Maps</FormLabel>
             <Accordion type="single" collapsible className="max-w-lg">
-              {maps.map((map) => (
+              {maps.map((map, index) => (
                 <AccordionItem key={map.id} value={map.id.toString()}>
                   <AccordionTrigger>{map.name}</AccordionTrigger>
                   <AccordionContent>
+                    <FormField
+                      control={form.control}
+                      name={`maps.${index}.replayCode`}
+                      render={({ field }) => (
+                        <FormItem>
+                          <FormLabel>Replay Code for {map.name}</FormLabel>
+                          <FormControl>
+                            <Input
+                              placeholder="Replay Code"
+                              className="w-28"
+                              {...field}
+                            />
+                          </FormControl>
+                          <FormMessage />
+                        </FormItem>
+                      )}
+                    />
```

Comment:
I think this UI could look a little nicer like this instead of stacked:
![CleanShot 2024-08-22 at 23 40 13@2x](https://github.com/user-attachments/assets/e3e18bde-1d55-4cff-9c0f-34aa7b968a16)
See the following implementation:
```tsx
<AccordionContent>
  <div className="flex items-center justify-between">
    <FormField
      control={form.control}
      name={`maps.${index}.replayCode`}
      render={({ field }) => (
        <FormItem>
          <FormLabel>Replay Code for {map.name}</FormLabel>
          <FormControl>
            <Input
              placeholder="Replay Code"
              className="w-28"
              {...field}
            />
          </FormControl>
          <FormMessage />
        </FormItem>
      )}
    />
    <Separator orientation="vertical" />
    <AlertDialog>
      <AlertDialogTrigger asChild>
        <Button className="mt-3" variant="destructive">
          Delete Map
        </Button>
      </AlertDialogTrigger>
      <AlertDialogContent>
        <AlertDialogHeader>
          <AlertDialogTitle>
            Are you absolutely sure?
          </AlertDialogTitle>
          <AlertDialogDescription>
            This will permanently delete the map{" "}
            <strong>{map.name}</strong>. This action cannot be
            undone.
          </AlertDialogDescription>
          <AlertDialogFooter>
            <AlertDialogCancel>Cancel</AlertDialogCancel>
            <AlertDialogAction
              onClick={() => deleteMap(map.id)}
            >
              {loading ? (
                <>
                  <ReloadIcon className="mr-2 h-4 w-4 animate-spin" />{" "}
                  Deleting...
                </>
              ) : (
                "Delete"
              )}
            </AlertDialogAction>
          </AlertDialogFooter>
        </AlertDialogHeader>
      </AlertDialogContent>
    </AlertDialog>
  </div>
```